packages/api-v2/test/major/major.e2e-spec.ts

```diff
@@ -2,6 +2,9 @@ import { INestApplication } from "@nestjs/common";
 import * as request from "supertest";
 import { initializeApp } from "../../test/utils";
 
+jest.useRealTimers();
+jest.setTimeout(50000);
```

Comment:
Is there a reason why we include a timeout in the beginning of everything?

Reply:
Works without it now, think there was a step in my debugging where I thought it was necessary lol